tests/Serializer/PropertyNormalizerTest.php

```diff
@@ -32,12 +40,20 @@ public function testCreateChildContext(): void
      * Tests createChildContext when no NameConverter exists
      *
      * @return void
+     *
+     * @throws \ReflectionException
      */
     public function testCreateChildContextWithoutNameConverter(): void
     {
-        $normalizer = new PropertyNormalizerStub();
+        $normalizer = new PropertyNormalizer();
+
+        // There is no simple way to test this method call, so lets use the Reflection approach.
+        // Bad idea, do not copy.
```

Comment:
Why is Reflection a better solution than using the Stub, but exposing the protected method via a public one in the stub?

Reply:
Because the class is final

Reply:
Extremely valid point is valid.